sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/ControlFlowGraphTest.cs

```diff
@@ -2807,7 +2807,7 @@ public void Cfg_Switch_Patterns_Default()
 
         [TestMethod]
         [TestCategory("CFG")]
-        public void Cfg_Switch_Patterns_When()
+        public void Cfg_Switch_Patterns_Two_Case_When()
```

Comment:
@duncanp-sonar - do you reckon there is some `switch` scenario (pre-C#8) which is not covered by the UTs?